features/search/works_stats.feature

```diff
@@ -85,7 +85,6 @@ Feature: Search works by stats
 
   # This is basically the same scenario as above, but the new search has
   # changed the "Complete" checkbox into a "Only complete works" radio button,
```

Comment:
Very minor: this should end in a period now.

Reply:
:+1: 

re: #3281 
haha, yea, it's probably because that one was still vaguely in memory that I took notice of the `* Or "Works in Progress"` comment.

To confirm - so original "Only incomplete works" was represented as "Incomplete" in the summary. New version "Works in progress only" should be represented as "Works in Progress" in the summary?

Since the naming relies on the other pull, I guess for this pull I'll keep the radio button selecting steps the same (`And I choose "Only incomplete works"`, etc.), and then later on depending on whichever pull makes it in first, I/whoever can go back and fix what will become a failing step.

Reply:
Someone else will confirm the new wording, but waiting for the other pull sounds good.